ubuntu/java11/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM dwolla/jenkins-agent-nucleus AS nucleus
+FROM eclipse-temurin:11
```

Comment:
I noticed that we're pinning to a specific JDK build in the Java 8 images. Should we do that for the Java 11 images, too, so that our builds are repeatable over time? (e.g. in this case, I think it would be `eclipse-temurin:11.0.14.1_1-jdk` instead of `eclipse-temurin:11`)

Reply:
We can make that change in a followup PR, though. We're going to build off this PR to implement multi-architecture variants of these images, so the pinning can be done as part of that subsequent change.